sloth.py

```diff
@@ -563,6 +563,8 @@ def generateSet(self, path, setname = None, cutextension = None):
 
 			if ext == self.slothFileExt:
 				slothfiles.add(mapname)
+			if ext.lower() not in [ ".bmp", ".jpg", ".jpeg", ".png", ".tga" ]:
```

Comment:
I think I would prefer if extensions weren't hardcoded but set to whatever PIL's `Image.open` can handle. (If there is no such list of extensions, you could also invoke `Image.open` and catch and warn about any exceptions.)